src/mds/Beacon.h

```diff
@@ -78,18 +79,21 @@ class Beacon : public Dispatcher
 
   bool is_laggy();
   double last_cleared_laggy() const {
-    Mutex::Locker l(lock);
+    std::unique_lock lock(mutex);
```

Comment:
@batrick It doesn't like this line, but I don't see what's wrong with it?

Reply:
ah, I had a stray `std::string_view` . . . maybe that was the problem

Reply:
hm, maybe **change this to `std::unique_lock<std::mutex> lock(mutex)` ?**

Reply:
Ah, sorry, I have not noticed that we changed `Mutex` type to `std::mutex` for the lock.

Reply:
@smithfarm So, have you tried `std::unique_lock<std::mutex> lock(mutex)`?